src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,53 @@
+@import "./utils/variables.scss";
+
+.nav {
+  display: flex;
+  align-items: center;
+
+  &__list {
+    display: flex;
+    justify-content: space-between;
+    margin: $navElementHeight 0;
+    padding-left: 0;
+    font-size: 0;
+  }
+
+  &__item {
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    text-transform: uppercase;
+    color: $color-main-accent;
+    text-decoration: none;
+    transition: $transitionTime;
+    padding: $navElementHeight 0;
+    position: relative;
+    white-space: nowrap;
+
+    &:hover {
+      transition: $transitionTime;
+      color: $color-accent-blue;
+    }
+  }
+}
+
+.is-active {
```

Comment:
Regarding your question about placing the `.is-active` class: it is better not to put (nested) it in other classes, but to keep it in the same file as the class for which you want to use it.
So you did everything right 👍